crossenv/__init__.py

```diff
@@ -397,10 +406,26 @@ def create_configuration(self, context):
             sysname = host_info[0]
             machine = host_info[-1]
 
+        release = ''
+        if self.macosx_deployment_target:
+            major, minor = self.macosx_deployment_target.split(".")
```

Comment:
This is nitpicky, but I would move this line into the try block as well.

Reply:
Thanks. Happy to fix it.